nb2/service/person_service.py

```diff
@@ -81,12 +127,57 @@ def create_person(data: CreatePersonDTO):
             f"Can't add Quote to Person with these fields empty: {empty_required_fields}"
         )
 
+    slack_user_id = getattr(data, "slack_user_id", None)
+    ghost_user_id = getattr(data, "ghost_user_id", None)
```

Comment:
This will throw the integrity error if ghost_user_id=None when committed right? Just confirming I understand

Reply:
i don't think it'll throw an error because the field isn't set to be required on the model side, should we make it required?

Reply:
I should have read the doc string. It specifically mentions throwing the Integrity error

Reply:
Only when there's someone with the same slack_user_id though (and `ghost_user_id`, will add to docstring). If you leave it as empty I don't think it errors.

Reply:
Are you suggesting making ghost_user_id nullable=False? If so then I think we should because we always want to be able to use ghost_user_id

Reply:
Changed the field to required and updated the migration file